drivers/scheduler/k8s/k8s.go

```diff
@@ -70,6 +70,22 @@ const (
 	resizeSupportedAnnotationKey = "torpedo/resize-supported"
 )
 
+const (
+	//PortworxStorage portworx storage name
+	PortworxStorage = "portworx"
+	//CsiStorage csi storage name
+	CsiStorage      = "csi"
+	portworxStorage = "portworx"
+	csiStorage      = "csi"
+	secretName      = "openstorage.io/auth-secret-name"
+	secretNamespace = "openstorage.io/auth-secret-namespace"
+)
+
+var provisioners = map[string]string{
```

Comment:
this isn't being reference anywhere. Remove.

Reply:
Fixed.